Add slack-logger example.

Hazealign opened the pull request:
Here's simple example for cottage.

therne commented:
All looks good, but some convention problem:
- use `const` instead of `let` when importing modules 
- prototype extending of builtin types is not good
- prefer explicit version dependencies in `package.json`
- `getFilePath` function is unnecessary (because it used only once)  
- I think we don't have to log body in sample code 
- I think slack token should be replaced to empty placeholder
It's not a big problem, but I think we can take more standard approach 👍 

and some questions:
- why did you put static serving middleware and 404 handler?

Hazealign commented:
I put static serving middleware for just viewing logs. 

 -  I changed `const` instead of `let` when importing modules
 - I removed extending builtin types by prototype
 - I removed unusing codes
 - And Slack token is now empty string. (It's just some convenience for setting token.)

Plz check it. :)

therne commented:
ok, looks great
